src/execution/execution.types.ts

```
@@ -0,0 +1,15 @@
+  outputFormat: string;
+  threshold: number;
+  useSemanticScoring: boolean;
+  // Define the properties needed for execution configuration
+  taskType: string;
+  parameters: Record<string, any>;
+}
+
+export interface ExecutionResult {
+  // Define the properties for the result of an execution
+  success: boolean;
+  output: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)